Reference/Scheduling/index.md

```diff
@@ -0,0 +1,90 @@
+---
+versionFrom: 8.0.0
+---
+
+# Scheduling with BackgroundTaskRunner
+In version Umbraco 8+ it is possible to run re-occuring code using the `BackgroundTaskRunner`.
+Below is a complete example showing how to register the Task with the runner with a component that will regularly empty out the recycle bin every five minutes.
```

Comment:
```suggestion
Below is a complete example showing how to register a Task with the runner with a component that will regularly empty out the recycle bin every five minutes.
```
Will it empty out the recycle bin every five minutes, or regularly? 🤔

Reply:
Yes every five minutes, I know a bit over the top for an example but gives people some inspiration/ideas on what they could do hopefully @sofietoft